filter_plugins/oo_filters.py

```diff
@@ -1,14 +1,15 @@
 #!/usr/bin/python
 # -*- coding: utf-8 -*-
 # vim: expandtab:tabstop=4:shiftwidth=4
+# pylint: disable=missing-docstring, no-self-use, too-few-public-methods
```

Comment:
We need a comment saying why these are being disabled and if it's mean to be temporary or permanent.